grumphp.yml

```diff
@@ -0,0 +1,26 @@
+grumphp:
+    fixer:
+        enabled: true
+        fix_by_default: true
+    tasks:
+        phpmd:
+            whitelist_patterns: [ ]
```

Comment:
The fields that you are not going to adjust anything I would say can be deleted if you want